### PR TITLE
[RAPTOR-7858] create and use base dropin env image with mlops

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

### First step
Create and publish a base env image
datarobot/dropin-env-base:debian10-py3.7-jre11-drum1.8.0-mlops8.0.7

Make python/java envs use that image.

This is the first step. Cleaning up other envs is in progress

### CODE REVIEW

File: docker/dropin_env_base/build_image.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (zohar-mizrahi) [inline docker/dropin_env_base/build_image.sh:1]: Much better to use a system agnostic approach:
```suggestion
#!/usr/bin/env bash
```

File: docker/dropin_env_base/build_image.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+GIT_ROOT=$(git rev-parse --show-toplevel)
+TEST_DOCKER_DIR=$GIT_ROOT/docker/drum_integration_tests_base
+
+IMAGE_NAME=datarobot/dropin-env-base
+IMAGE_TAG=debian10-py3.7-jre11-drum1.8.0-mlops8.0.7
+
+pwd
+
+echo "Building docker image: ${IMAGE_NAME}:${IMAGE_TAG}"
+DATAROBOT_MLOPS_VERSION=8.0.7 ./pull_artifacts.sh
```
Comment (zohar-mizrahi) [inline docker/dropin_env_base/build_image.sh:18]: Better to use a non relative path, by adding the script folder to the path:
`script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"`

File: docker/dropin_env_base/pull_artifacts.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (zohar-mizrahi) [inline docker/dropin_env_base/pull_artifacts.sh:1]:
```suggestion
#!/usr/bin/env bash
```

File: docker/dropin_env_base_jdk/build_image.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (zohar-mizrahi) [inline docker/dropin_env_base_jdk/build_image.sh:1]:
```suggestion
#!/usr/bin/env bash
```

File: docker/dropin_env_base_jdk/build_image.sh [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+GIT_ROOT=$(git rev-parse --show-toplevel)
+TEST_DOCKER_DIR=$GIT_ROOT/docker/drum_integration_tests_base
+
+IMAGE_NAME=datarobot/dropin-env-base
+IMAGE_TAG=debian10-py3.7-jdk11-drum1.8.0-mlops8.0.7
+
+pwd
+
+echo "Building docker image: ${IMAGE_NAME}:${IMAGE_TAG}"
+DATAROBOT_MLOPS_VERSION=8.0.7 ./pull_artifacts.sh
```
Comment (zohar-mizrahi) [inline docker/dropin_env_base_jdk/build_image.sh:18]: Same here - use non-relative path

**Review by zohar-mizrahi — CHANGES_REQUESTED** [review]
I wonder for the JDK base env - would it make sense to use soft link to the files in the JRE where possible?

File: docker/dropin_env_base/pull_artifacts.sh (hunk 3 above)
Comment (yakov-g) [reply docker/dropin_env_base/pull_artifacts.sh:1]: DONE

File: docker/dropin_env_base_jdk/build_image.sh (hunk 4 above)
Comment (yakov-g) [reply docker/dropin_env_base_jdk/build_image.sh]: DONE

File: docker/dropin_env_base_jdk/build_image.sh (hunk 5 above)
Comment (yakov-g) [reply docker/dropin_env_base_jdk/build_image.sh]: DONE

**Review by meltzg — APPROVED** [review]

**Pull request merged**